final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,127 @@
+"""Main module of the program"""
+
+import sys
+import os
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(this_directory)
+
+
+import logs
+import args_parser
+import rss_parser
+import other
+import json_converter
+import news
+
+
+def main():
+
+    """This is a main function"""
+    print('============================== RSS Reader ==================================')
+    logs.new_session()
+    commands = args_parser.get_parse()
+
+    while True:
+        while True:
```

Comment:
A question:

Is that really necessary to use 2 cycles here?

Reply:
Two cycles will be used for advanced news storage, because of it I need two separate "while" cycles inside one general "while" cycle. It will be done during the next commit.

Reply:
It is still strongly recommended to avoid such constructions inside production code. Please try to think of some other way to run your application

Reply:
Fixed.